asciinema/README.md

```diff
@@ -0,0 +1,11 @@
+This directory includes the configuration for rendering the asciinema
+panel in the tekton.dev homepage.
+
+A rendered panel (an asciinema cast) has been included at
+`../static/asciinema/demo.cast`. To rebuild the cast, run the commands below:
+
+```bash
+./run.sh
+rm -f ../static/asciinema/demo.cast
+mv ./demo.cast ../static/asciinema/demo.cast
+```
```

Comment:
does asciienma work with netlify? Im thinking maybe the Makefile being used by netlify will need an update?

Reply:
For this part, since we probably do not need to update the asciinema cast very often, this script is not executed at the time of deployment. A finished cast is already checked in with this PR, which will be uploaded as a static file :)